hack/make.sh

```diff
@@ -44,6 +44,7 @@ echo
 DEFAULT_BUNDLES=(
 	validate-dco
 	validate-gofmt
+    validate-vet
```

Comment:
Looks like the indentation is "off" here; possibly mixed tabs/spaces?

(Not urgent, you may want to wait until after this has been reviewed)